web/app.js

```diff
@@ -565,6 +566,14 @@ const PDFViewerApplication = {
       this.findBar = new PDFFindBar(appConfig.findBar, eventBus, this.l10n);
     }
 
+    if (annotationEditorEnabled) {
+      const editorModeButtons = appConfig.toolbar.editorNoneButton.parentNode;
+      editorModeButtons.classList.remove("hidden");
+
+      const editorModeSeparator = editorModeButtons.nextElementSibling;
```

Comment:
This node should have an `id` and we should check that the `id` is correct (and display an error in case something is wrong).